Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -147,6 +148,11 @@ namespace EMotionFX
         }
     }
 
+    void ColliderPropertyNotify::AfterPropertyModified([[maybe_unused]] AzToolsFramework::InstanceDataNode* pNode)
```

Comment:
nit: I'd drop the Hungarian notation `p` prefix, so this would just be `node`

Reply:
That's just the existing style that was in the file already.